### DISCUSSION TITLE
Long time to pick up new shows from Radarr / Jellyfin

### DISCUSSION THREAD

**Discussion by alvitali** [seerr-team/seerr#885] [Q&A] [answered]
Jellyseerr sometimes takes pretty long to pick up shows which have been made available for some time already by Radarr, and picked up in Jellyfin (e.g. a show added to Jellyfin's logs at 22:37, but only picked up by Jellyseerr at 03:10).

Jellyseerr (sometimes) only seems to register these shows during the "Manual library scan", run every 24 hours. Is this expected behaviour? And is there a way to poll newly added items more often, or even have Radarr or Jellyfin push those infos to Jellyseerr right away?

Thank you for your work on Jellyseerr, in any case.

**Comment by fallenbagel** [comment, marked as answer]
This is a configuration issue.

Basically, your date added registered by jellyfin is not recent enough for it to pass it to jellyseerr to be picked up by recently added scan.

You can fix this by going to jellyfin dashboard > library > display and change date added to `date scanned into library`. Jellyfin uses date added to determine what's recently added and what's not.

(p.s for support issues, please use our discord as that's where we carry out support.)

**Reply by alvitali** [reply]
Thanks a lot! 

Sorry, will pop over to the Discord next time.
